src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -972,11 +973,19 @@ class ContentSidebar extends PureComponent<Props, State> {
      * @return {void}
      */
     fetchCurrentUser(user?: User, shouldDestroy?: boolean = false): void {
+        const { fileId = '' } = this.props;
         if (shouldRenderSidebar(this.props)) {
             if (typeof user === 'undefined') {
+                const url = this.api.getUsersAPI(false).getUrl();
```

Comment:
You should not bleed api stuff in here as much as possible. Otherwise we are replicating some internals here.
Change the get function to to take an id and provide it to xhr instead.

Reply:
you're right, I switched up the Users API since it's the only endpoint that's base getURL won't consider an ID